Src/Plugins/Security/Utilities/StaticStringHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities
+{
+    public static class StaticStringHelper
```

Comment:
>StaticStringHelper [](start = 24, length = 18)

Here's a rule for you: please don't create any static helper classes with a single method in it. Instead, create a single file named Utilities to hold diverse helpers until there's enough of them to warran grouping/refactoring.

Reply:
Create a file named 'FilteringHelpers'. Move this code into it and delete this file. Move all DomainFilteringHelper code into and delete that file. Thanks. Once you have had the experience of watching years of file/class/etc. profileration lead to a code base that is unwieldy and hard to manage, you may begin to agree with me that this is a worthwhile concern. For now, I don't require you to be persuaded of that. :) But please do make the change, I appreciate it.

---
In reply to: [611237162](https://github.com/microsoft/sarif-pattern-matcher/pull/372#discussion_r611237162) [](ancestors = 611237162)

Reply:
Done.

---
In reply to: [611237758](https://github.com/microsoft/sarif-pattern-matcher/pull/372#discussion_r611237758) [](ancestors = 611237758,611237162)